chore: add Sapphire to the list of useful servers (#17)
discordjs/resource-webhooks: resources/USEFUL_SERVERS.md

> https://discord.gg/arTauDY

_ _
**• Sapphire Support**

> > Want to find out how to make the next big bot using Sapphire?

Reviewer: get your own slogan 🙄

Reviewer 2: ```suggestion
> Want to find out how to make the next even-bigger bot using Sapphire?
```

Nah, just kidding. It's hard to come up with something... What about...

- Sapphire is a fully customizable framework which you can use as a robust base for your next Discord bot!
- Sapphire is a low-level framework with first-class TypeScript support, designed to be customizable to its core.
- Sapphire is the framework to go if you need a very fast and powerful command handler, want to start?

I can't think of many things except this framework is a little of two (the Piece/Store is inspired on Klasa with extra abstractions such as strategies, and dynamic argument parsing from Akairo, powered by Lexure, made by the same developer) and a bit of our own (typed named arguments, easy callback-based custom arguments...) plus things we learnt from others (`Result<T, E>` and `Maybe<T>` are used in all of the framework's core).

Author: A mix of 1 and 2 would be ideal.... Time to pull out the thinking cap!

Reviewer 2: The thinking cap came up with this:
```suggestion
> Sapphire is a fully customizable framework with first-class TypeScript support you can use as a robust base for your next Discord bot!
```

> Join the official support server for the Sapphire framework!
> https://discord.gg/j8ACK6u

> - -

Reviewer 2: ```suggestion
_ _
```
